internal/org/repository/view/org_member.go

```diff
@@ -0,0 +1,99 @@
+package view
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/caos/logging"
+	caos_errs "github.com/caos/zitadel/internal/errors"
+	"github.com/caos/zitadel/internal/eventstore/models"
+	"github.com/caos/zitadel/internal/org/model"
+	"github.com/lib/pq"
+)
+
+const (
+	OrgMemberKeyUserID    = "user_id"
+	OrgMemberKeyOrgID     = "org_id"
+	OrgMemberKeyUserName  = "user_name"
+	OrgMemberKeyEmail     = "email"
+	OrgMemberKeyFirstName = "first_name"
+	OrgMemberKeyLastName  = "last_name"
+)
+
+type OrgMemberView struct {
+	UserID    string         `json:"userId" gorm:"column:user_id;primary_key"`
+	OrgID     string         `json:"-" gorm:"column:org_id;primary_key"`
+	UserName  string         `json:"-" gorm:"column:user_name"`
+	Email     string         `json:"-" gorm:"column:email_address"`
+	FirstName string         `json:"-" gorm:"column:first_name"`
+	LastName  string         `json:"-" gorm:"column:last_name"`
+	Roles     pq.StringArray `json:"roles" gorm:"column:roles"`
```

Comment:
i think it should be 'json:"role_keys" ... but i'm not sure

Reply:
we only use `roleKeys` if the object is granted (`user_grant`, `granted_project`)